webapp/lib/controller.dart

```diff
@@ -587,19 +586,19 @@ void command(UIAction action, Data data) {
       platform.updateUnread([activeConversation], false).catchError(showAndLogError);
       break;
     case UIAction.markConversationUnread:
-      if (multiSelectMode) {
-        var markedConversations = <model.Conversation>[];
-        for (var conversation in selectedConversations) {
-          if (!conversation.unread) {
-            markedConversations.add(conversation);
-            view.conversationListPanelView.markConversationUnread(conversation.docId);
-          }
-        }
-        platform.updateUnread(markedConversations, true).catchError(showAndLogError);
-      } else {
+      if (!currentConfig.sendMultiMessageEnabled || selectedConversations.isEmpty) {
         view.conversationListPanelView.markConversationUnread(activeConversation.docId);
         platform.updateUnread([activeConversation], true).catchError(showAndLogError);
```

Comment:
Would it make sense for `selectedConversations` to have the following characteristics?
* always non-null
* always contain `activeConversation` if `activeConversation != null`
* always be empty if `activeConversation == null`

If this were true, then several action handlers like this one could be simplified.

Reply:
Thanks @danrubel I'll need to have a think in more detail about this as the separation between them seems helpful - in multiConversationSelect mode, the active conversation is just an indicator for the conversation you're seeing in the second panel which supports a workflow where the user goes through the list, reads each conversation, and only selects it if they want to add it to a bulk reply list. Always considering `activeConversation` as part of the `selectedConversations` would require changing that workflow or doing something else in the UI to support it.